packages/engine/src/assets/classes/AssetLoader.ts

```diff
@@ -197,6 +200,55 @@ export class AssetLoader {
     load(params, onLoad, onProgress, onError)
   }
 
+  static async loadInstancedGLTF(
```

Comment:
Can this just return promise instead of accepting an `onLoad` callback arg _and_ returning a promise? Like what [this guy](https://github.com/XRFoundation/XREngine/pull/3910/files#diff-0101def9fedb24e71b1585b997ad39b8f4fe0bf91c09a332d8e1ad194cf0bda3R83) is doing.

Reply:
Also this seems almost the same as that function. Why both?